client/components/status/status.js

```diff
@@ -133,9 +135,10 @@ export default class Status extends React.Component {
             this.loginFormRef &&
             this.loginFormRef.current &&
             justAuthenticated
-          )
+          ) {
             this.loginFormRef.current.submit();
-
+            sessionStorage.removeItem(`${orgSlug}_just_authenticated`);
```

Comment:
this test does fail without the fixes but the message shown (`should store token in sessionStorage when remember me is unchecked and rememberMe in localstorage`) doesn't reflect the real issue.
That's why I suggest ensuring the other test is the one that should alert us if anything goes wrong with the captive portal login feature.